### PR TITLE
Fix PATH for cargo cache

### LINKED ISSUE #2749: Moving .cargo to DevDrive should also adjust PATH
### Dev Home version

0.1301.477.0

### Windows build number

10.0.22631.3296

### Other software

OS Build Version: 10.0.22631.3296.amd64fre.ni_release.220506-1250
.NET Version: .NET 8.0.4

### Steps to reproduce the bug

1. Use the option to move the .cargo dir
2. Try to use cargo/rustc/rustup

### Expected result

cargo/rustc/rustup should work

### Actual result

The term 'cargo' is not recognized as a name of a cmdlet, function, script file, or executable program.

PATH still points to old .cargo/bin

### Included System Information

CPU: Intel(R) Core(TM) i5-9300H CPU @ 2.40GHz
Physischer Speicher: 15.86GB (4.19GB free)
Prozessorarchitektur: x64

### Included Extensions Information

Erweiterungen: 
Microsoft.Windows.DevHome_0.1301.477.0_x64__8wekyb3d8bbwe (Dev Home (Vorschau))
Microsoft.Windows.DevHome_0.1301.477.0_x64__8wekyb3d8bbwe (Dev Home (Vorschau))

Widgetdienst: 
MicrosoftWindows.Client.WebExperience_424.1301.450.0_x64__cw5n1h2txyewy

### PR DESCRIPTION
**Pull request by SohamDas2021**
## PR checklist
- [ ] Closes #2749
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs [hunk 1]
```diff
@@ -153,6 +153,50 @@ private void SetEnvironmentVariable(string variableName, string value)
         try
         {
             Environment.SetEnvironmentVariable(variableName, value, EnvironmentVariableTarget.User);
+
+            if (string.Equals(variableName, "CARGO_HOME", StringComparison.OrdinalIgnoreCase))
```
Comment (krschau) [inline tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs:157]: Will this only ever apply to CARGO_HOME? If not (and maybe even if so, for readability) suggest refactoring this new code into a method that can be reused.

Comment (SohamDas2021) [reply tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs:157]: This only applies to CARGO_HOME. But, yes, I will refactor this into a function.

**Review by krschau — APPROVED** [review]

File: tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs [hunk 2]
```diff
@@ -17,6 +17,7 @@
 using Windows.Media.Protection;
 using Windows.Storage.Pickers;
 using WinUIEx;
+using YamlDotNet.Core.Tokens;
```
Comment (manodasanW) [inline tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs:20]: Did I miss what we are using from this new namespace?

**Review by manodasanW — APPROVED** [review]

File: tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs (hunk 2 above)
Comment (SohamDas2021) [reply tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/OptimizeDevDriveDialogViewModel.cs]: I deleted this in local. Was waiting for comments before pushing out

**Pull request merged**